src/Depends.Core/DependencyAnalyzer.cs

```diff
@@ -183,91 +183,103 @@ public DependencyGraph Analyze(string projectPath, string framework = null)
                 // Todo: Something went wrong, log and return better exception.
                 throw new InvalidOperationException("Unable to load project.");
             }
-
-            if (!analyzerResult.IsNetSdkProject())
-            {
-                // Todo: Support "legacy" projects in the future.
-                throw new InvalidOperationException("Unable to load project.");
-            }
+            var projectNode = new ProjectReferenceNode(projectPath);
+            var builder = new DependencyGraph.Builder(projectNode);
 
             var projectAssetsFilePath = analyzerResult.GetProjectAssetsFilePath();
 
             if (!File.Exists(projectAssetsFilePath))
             {
-                // Todo: Make sure this exists in future
-                throw new InvalidOperationException($"{projectAssetsFilePath} not found. Please run 'dotnet restore'");
-            }
+                if (analyzerResult.IsNetSdkProject())
+                {
+                    // a new project doesn't have an asset file
+                    throw new InvalidOperationException("Unable to load project.");
```

Comment:
`throw new InvalidOperationException($"{projectAssetsFilePath} not found. Please run 'dotnet restore'");`